ponyo/vae.py

```diff
@@ -106,7 +105,7 @@ def tybalt_2layer_model(
     # The below is necessary for starting core Python generated random numbers
     # in a well-defined state.
 
-    rn.seed(12345)
+    np.random.seed(12345)
```

Comment:
You're setting the numpy seed twice in this file, and the seed for the base `random` package is no longer set. The second thing may not matter since you're not importing random in this file, but the first one appears to be a copy number variation